modules/ROOT/pages/bonita-bpm-overview.adoc

```diff
@@ -1,81 +1,94 @@
+[.card.card-index]
+--
+xref:ui-designer-overview.adoc[[.card-title]#Bonita UI Designer# [.card-body.card-content-overflow]#pass:q[Create the User Interfaces of you applications]#]
+--
 
-== Getting started
+[.card.card-index]
+--
+xref:bonita-studio-download-installation.adoc[[.card-title]#Studio installation# [.card-body.card-content-overflow]#pass:q[First step to use Bonita Studio]#]
+--
 
-*I'm currently using Bonita 6.x. Will 7.x be a big change?*
+Bonita Studio also embeds a *Bonita Runtime* (Tomcat, UI Designer, Bonita provided applications, Bonita Engine, and an h2 database), suitable for testing an application that is in development. When you run a process, it is automatically deployed onto the development platform.
 
-Bonita 7.x provides new and improved features, including the xref:ui-designer-overview.adoc[UI Designer] for creating application pages and forms, as well as xref:contracts-and-contexts.adoc[contracts and context] to create a clear split between the process logic, the data, and the user views.
+Then, once a process is ready, you can xref:build-a-process-for-deployment.adoc[build] it and deploy it on your Bonita qualification or production Runtime environment. +
 
-You will also find lots of familiar features, which continue to work as they did in 6.x.
+[#platform]
 
-*Tell me about the documentation.*
+== Bonita Runtime
 
-We're continually updating the documentation. +
-There are four main information categories: Application and process design, Installation, Portal administration, and Development.
+Bonita Runtime is composed of one or more Bonita Server nodes installed on some hosts (physical machine, virtual machine, a cloud instance, a container like Docker, ...). A Bonita Runtime composed of multiple nodes constitutes a cluster. +
 
-*Tell me about using Bonita.*
+Bonita Runtime is where the Bonita provided applications are located. +
 
-See xref:lifecycle-and-profiles.adoc[Lifecycle and profiles].
+As mentioned before, the Bonita User Application is the part of Bonita that is visible to process users, who use it to view tasks and take actions thanks to a generic task list for all processes they are involved in. +
 
-*Can I migrate an existing process into Bonita 7.x?*
+Bonita also comes with three very useful applications:
 
-You can xref:import-and-export-a-process.adoc[import a process] from any earlier Bonita release if  they don't contain Legacy v6 forms or case overview pages. If they do, you need to rework your process and xref:migrate-a-form-from-6-x.adoc[migrate those froms or pages].
+* Bonita Administrator Application: the tool used by the tenant administrator to xref:processes.adoc[install, deploy and manage processes] and to xref:applications.adoc[build applications]. We use the word "tenant" here, in case the platform would be made of xref:multi-tenancy-and-tenant-configuration.adoc[several tenants]. 
```

Comment:
Are you sure we want to encourage users to build applications from the Admin application? 
I think that it would be better here to talk about managing the organization, the resources and the access control.

Reply:
Totally agree. This is old content to update. Changing that.